eng/scripts/Deploy-TestResources.ps1

```diff
@@ -32,24 +34,26 @@ if(!$ResourceGroupName) {
 
 Push-Location $RepoRoot
 try {
-    $armParameters = @{ areas = ($Areas ?? @()) }
+    $armParameters = @{ areas = $Areas }
```

Comment:
I think this may be unintentional.  If someone calls the script with `-Areas $null` we still want to populate the `areas` arm parameter with an empty array

Reply:
```
> function something([string[]]$Areas){ @{areas=$Areas} | ConvertTo-Json -Compress }

> something -Areas $null
{"areas":null}
```